### PR TITLE
Improvement: use x86_64 Unity for both Apple Silicon and Intel processors

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**
It's now possible to install and run Unity on Apple Silicon Macs

### CODE REVIEW

File: content/knowledge-others/install-unity-version.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ aliases:
  - /knowledge-base/install-unity-version
 ---
 
-Each build machine image has a specific version of Unity installed. You can find out the specific Unity version by consulting the build machine specification for [macOS](../specs/versions-macos) and for [Windows](../specs/versions-windows) instances.
+Some build machines have Unity pre-installed. Consult build machine specification ([macOS](../specs/versions-macos) and [Windows](../specs/versions-windows)) to find out the version of the pre-installed Unity if any.
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:9]:
```suggestion
```

File: content/knowledge-others/install-unity-version.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ aliases:
...
 
 ## Use Codemagic's supported versions
 
-If you need to use a different Unity version, then you can specify this version in your `codemagic.yaml` file like this:
+If you need to install or change Unity version, you would need to specify the preferred version in your `codemagic.yaml` file like this:
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:13]:
```suggestion
If you need to install Unity, you would need to specify the preferred version in your `codemagic.yaml` file like this:
```

File: content/knowledge-others/install-unity-version.md [hunk 3]
```diff
@@ -56,6 +56,7 @@ If you can't find your desired Unity version in the list, please contact us [her
 {{<notebox>}}
 **Notes:**
 - These versions are the `Unity Editor (macOS x86_64)`, and have only the `macOS`, `Android`, and `iOS` modules.
+- Mac machines with Apple Silicon support Unity built for `x86_46` architecture.
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:59]:
```suggestion
- Mac machines with Apple silicon support Unity built for `x86_46` architecture.
```

**Review by mikhail-tokarev — CHANGES_REQUESTED** [review]

**Review by helinanever — APPROVED** [review]
LGTM

**Comment by helinanever** [comment]
As I've learned, specifying the Unity version in yaml will not work for Windows. There should be a note about this in docs.

File: content/knowledge-others/install-unity-version.md [hunk 4]
```diff
@@ -6,11 +6,11 @@ aliases:
  - /knowledge-base/install-unity-version
 ---
 
-Each build machine image has a specific version of Unity installed. You can find out the specific Unity version by consulting the build machine specification for [macOS](../specs/versions-macos) and for [Windows](../specs/versions-windows) instances.
+Intel Mac and Windows machines have Unity pre-installed. It's possible to install other supported Unity versions.
 
-## Use Codemagic's supported versions
+## Use Codemagic's supported versions (macOS only)
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:11]:
```suggestion
## Quick install (macOS only)
```

File: content/knowledge-others/install-unity-version.md [hunk 5]
```diff
@@ -6,11 +6,11 @@ aliases:
  - /knowledge-base/install-unity-version
 ---
 
-Each build machine image has a specific version of Unity installed. You can find out the specific Unity version by consulting the build machine specification for [macOS](../specs/versions-macos) and for [Windows](../specs/versions-windows) instances.
+Intel Mac and Windows machines have Unity pre-installed. It's possible to install other supported Unity versions.
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:9]:
```suggestion
```

File: content/knowledge-others/install-unity-version.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ workflows:
 This will automatically install the specified Unity version to the build machine and set the `UNITY_HOME` to `/Applications/Unity/Hub/Editor/<YOUR-DESIRED-UNITY-VERSION>/Unity.app` and you can continue building your app as described [here](../yaml-quick-start/building-a-unity-app/).
 
 
-#### The supported Unity versions on Mac Pro (Intel) machines are the following:
+**The available Unity versions on Mac machines are the following:**
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:25]:
```suggestion
**The available Unity versions are the following:**
```

File: content/knowledge-others/install-unity-version.md [hunk 7]
```diff
@@ -56,6 +56,7 @@ If you can't find your desired Unity version in the list, please contact us [her
 {{<notebox>}}
 **Notes:**
 - These versions are the `Unity Editor (macOS x86_64)`, and have only the `macOS`, `Android`, and `iOS` modules.
+- Mac machines with Apple silicon support Unity built for `x86_46` architecture.
 - If your app requires additional modules then you need to install it using [Unity Hub CLI](./-others/install-unity-version/#unity-installation-script) like this: `/Applications/Unity\ Hub.app/Contents/MacOS/Unity\ Hub -- --headless install-modules --version <UNITY_VERSION> -m windows-mono`, this will install the windows modules for the specified` <UNITY_VERSION>`.
 {{</notebox>}}
```
Comment (mikhail-tokarev) [inline content/knowledge-others/install-unity-version.md:62]:
```suggestion
## Download and install from Unity Hub
```

Comment (mikhail-tokarev) [reply content/knowledge-others/install-unity-version.md:62]: don't apply just it. It should be for the next line

**Review by mikhail-tokarev — APPROVED** [review]
🚀

**Pull request merged**